LibraBFT/Impl/Consensus/ChainedBFT/RoundManager.agda

```diff
@@ -18,11 +18,11 @@ open import LibraBFT.Abstract.Types.EpochConfig UID NodeId
 
 -- This is a minimal/fake example handler that obeys the VotesOnce rule, enabling us to start
 -- exploring how we express the algorithm and prove properties about it.  It simply sends a vote for
--- 1 + its LatestVotedRound, and increments its LatestVotedRound.  It is called EventProcessor for
+-- 1 + its LatestVotedRound, and increments its LatestVotedRound.  It is called RoundManager for
 -- historical reasons, because this what a previous version of LibraBFT called its main handler;
 -- this will be updated when we move towards modeling a more recent implementation.
 
-module LibraBFT.Impl.Consensus.ChainedBFT.EventProcessor
+module LibraBFT.Impl.Consensus.ChainedBFT.RoundManager
```

Comment:
This should also be moved to `LibraBFT.Impl.Consensus.RoundManager`, keeping the structure consistent with the  implementation (`ChainedBFT` is obsolete just like `EventProcessor`!).

Reply:
Done.